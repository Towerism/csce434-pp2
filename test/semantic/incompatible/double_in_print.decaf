int asdf() {
    double a;
    Print(a);
}