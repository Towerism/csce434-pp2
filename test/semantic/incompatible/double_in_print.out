
*** Error line 3.
    Print(a);
          ^
*** Incompatible argument 1: double given, int/bool/string expected

